src/Framework/Telemetry/LoggingConfigurationTelemetry.cs

```diff
@@ -56,32 +56,16 @@ internal class LoggingConfigurationTelemetry : TelemetryBase
     /// </summary>
     public bool ConsoleLogger { get; set; }
 
-    /// <summary>
-    /// Type of console logger: serial | parallel
-    /// </summary>
-    public string? ConsoleLoggerType { get; set; }
-
     /// <summary>
     /// Verbosity of console logger: quiet | minimal | normal | detailed | diagnostic
     /// </summary>
     public string? ConsoleLoggerVerbosity { get; set; }
 
-
     /// <summary>
     /// True if file logger was used.
     /// </summary>
     public bool FileLogger { get; set; }
 
-    /// <summary>
-    /// Type of file logger: serial | parallel
-    /// </summary>
-    public string? FileLoggerType { get; set; }
-
-    /// <summary>
-    /// Number of file loggers.
-    /// </summary>
-    public int FileLoggersCount { get; set; }
```

Comment:
I don't get why it's changed by the pr. It's supposed to be about serial console logger but this relates to file logger.

Reply:
you're right, filelogger is derived from consolelogger so removing this data is correct